### PR TITLE
Added left over params

### PR DESCRIPTION
**Pull request by kontr0x**
+ added multi edit functionality

### CODE REVIEW

File: API/api/helperClasses/enums/validation.php [hunk 1]
```diff
@@ -5,12 +5,18 @@
...
      */
 
     abstract class Validation{
         const ApiKey = '/^[0-9a-zA-Z]{20}$/';
         const Session = '/^[0-9a-zA-Z]{20}$/';
         const UserName = '/^[0-9a-zA-Z]{3,20}$/';
+        const BooleanValue = '/^(?:True){1}$|^(?:False){1}$/';
+        const Overtime = '/^(?:[0-9]|[1-9][0-9]|[1-9][0-9][0-9])$/';
+        const WeeklyWorkingMinutes = '/^(?:[0-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-2][0-9][0-9][0-9]|[1-3][0]{3})$/';
+        const WeeklyWorkingDays = '/^(?:[0-6]{1})$/';
```
Comment (lululujojo123) [inline API/api/helperClasses/enums/validation.php]: I think weekly working days should go up to 7 days? There are some rare situations with Sunday working contracts. What do you think @TrickyTreat @an055?

**Review by lululujojo123 — COMMENTED** [review]
Apart from that one point I think everything should be okay. Thanks for making that changes.

File: API/api/helperClasses/enums/validation.php (hunk 1 above)
Comment (annNau) [reply API/api/helperClasses/enums/validation.php]: Yes, I think 7 days would be good. Then we can cover every possibility.

**Comment by kontr0x** [comment]
Changed the validation to a maximum of 7 working days a week. That should meet the requirements.

**Review by lululujojo123 — APPROVED** [review]
Thanks for doing the change. I'll merge and close that request and branch.

**Pull request merged**